app/controllers/api/v1/moves_controller.rb

```diff
@@ -115,6 +116,26 @@ def move
       def updater
         @updater ||= Moves::Updater.new(move, update_move_params)
       end
+
+      def validate_include_params
+        # TODO: this is temporary, once FE uses the `include` params for all the endpoints/resources,
+        # supported_attributes will be set to to MoveSerializer::SUPPORTED_RELATIONSHIPS
+        supported_attributes = MoveSerializer::SUPPORTED_RELATIONSHIPS + %w[profile]
```

Comment:
I think we can just return the profile and the person in the `MoveSerializer::SUPPORTED_RELATIONSHIPS` constant. The frontend will deserialize the json and not access the `profile` key so this is safe.

This means we can add:

```ruby
      class UnsupportedIncludeError < StandardError; end

      rescue UnsupportedIncludeError, with: :render_unsupported_includes

      def validate_include_params
        included_relationships.each do |relationship|
          raise UnsupportedIncludeError unless MoveSerializer::SUPPORTED_RELATIONSHIPS.include?(relationship)
        end
      end

      def included_relationships
        IncludeParamHandler.new(params).call || MoveSerializer::SUPPORTED_RELATIONSHIPS
      end

      def render_unsupported_includes(exception)
           render
                   status: :bad_request,
                   json: {
                     errors: [{ title: 'Bad request',
                                detail: "'#{resource}' is not supported. Valid values are: #{supported_attributes.join(', ')}" }],
                   }
      end
```

Notice I've renamed supported_attributes to supported_relationships and am propagating an error in the same way we will in the api base controller.

Reply:
And just copy and paste the propagated exception

Reply:
When we've migrated all of the includes

Reply:
i.e. to the base controller